quickstart/exec-htap-params.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Why another exec*.sh file?
Can the existing file be enhanced to cover needed functionality?
Can this file be added to that specific environment it is meant for?

Reply:
we can change the existing one, the key difference is the fact that the cloud team does not want to have ant installed on the image, so the new exec not only allows for parameters instead of editing config.sh, but also removes the build step. The idea is that the image will be created with the build already done. Would you prefer that I add the param functionality to regular exec and also put an "if" around the build step?

Reply:
If exec_htap.sh defines an interface between two teams it should be contracted. I would agree to a general mechanism of providing command line parameters to override those in config.sh (editing config.sh remains an option, of course) and to skip build (invoking ant). I don't agree  to the proposed rigid form I'm expected to maintain.